frontend/src/features/recommendation/SearchFilter.spec.tsx

```diff
@@ -10,8 +14,15 @@ import { dateChoices } from './DateFilter';
 import { languageChoices } from './LanguageFilter';
 import { mainCriteriaNames } from 'src/utils/constants';
 
+declare module '@mui/styles/defaultTheme' {
+  // eslint-disable-next-line @typescript-eslint/no-empty-interface
+  interface DefaultTheme extends Theme {}
+}
+
 describe('Filters feature', () => {
-  let pushSpy = null;
+  // TODO figure out the typing here
```

Comment:
There is still a TODO here and in Login.spec.tsx. But it doesn't matter if you fix it now or later.